systems_cross_building/cross_building.rst

```diff
@@ -128,8 +128,8 @@ You can find working examples at the :ref:`bottom of this section <cross_buildin
 
 .. _cross_building_build_requires:
 
-Using build requires
-++++++++++++++++++++
+Using tool requires
```

Comment:
Maybe I would also add the note saying that tool requires used to be build requires here...

Reply:
done